packages/wrangler/src/environment-variables/misc-variables.ts

```diff
@@ -4,22 +4,25 @@ import { getEnvironmentVariableFactory } from "./factory";
  * `WRANGLER_C3_COMMAND` can override the command used by `wrangler init` when delegating to C3.
  *
  * By default this will use `create cloudflare@2`.
- * If you want to test the integration between wrangler and C3 locally, you can do the following:
+ *
+ * To run against the beta release of C3 use:
+ *
+ * ```sh
+ * # Tell Wrangler to use the beta version of create-cloudflare
+ * WRANGLER_C3_COMMAND="create cloudflare@beta" npx wrangler init
+ * ```
+ *
+ * To test the integration between wrangler and C3 locally, use:
  *
  * ```sh
  * # Ensure both Wrangler and C3 are built
  * npm run build
- * # Run the command from the Wrangler package directory
- * cd packages/wrangler
  * # Tell Wrangler to use the local version of create-cloudflare
- * WRANGLER_C3_COMMAND="run create-cloudflare" npx wrangler init temp
+ * WRANGLER_C3_COMMAND="exec ./packages/create-cloudflare" npx wrangler init temp
  * ```
  *
- * This makes use of the fact we have a monorepo and npm will delegate to the local
- * version of a package if possible.
- * Since the way `wrangler init` is written, it will always use `npm` (or equivalent) to run C3, rather than `npx` (or equivalent).
- * Therefore we have added a `create-cloudflare` script to the Wrangler `package.json` that will work with `npm run create-cloudflare`.
- * But that means that you must run the `npx wrangler init` command from the `wrangler` package directory.
+ * Note that you cannot use `WRANGLER_C3_COMMAND="create cloudflare@2"` if you are
+ * running Wrangler from inside the monorepo as the bin paths get messed up.
```

Comment:
Should the `create-cloudflare` script be removed in that case?

Reply:
Yes it is removed.

Reply:
https://github.com/cloudflare/workers-sdk/pull/3483/files#diff-46fcb5c28c10835193722317df45b137b19a1c56c8e5d690fe8b9f2b1b685418L59